datadog/api/metrics.py

```diff
@@ -54,6 +54,12 @@ def send(cls, metrics=None, attach_host_name=True, **single_metric):
         :param metric: the name of the time series
```

Comment:
let's just describe more explicitly in the main description above what a metric dict should look like